frontend-react/src/utils/PermissionsUtils.ts

```diff
@@ -35,4 +35,18 @@ const permissionCheck = (
     }
 };
 
+export interface RSUserPermissions {
+    isUserAdmin: boolean;
+    isUserSender: boolean;
+    isUserReceiver: boolean;
+}
+
+export function getUserPermissions(user?: RSUserClaims): RSUserPermissions {
+    return {
+        isUserAdmin: !!user?.organization.find((o) => isAdmin(o)),
```

Comment:
Ah, I think `organization` is on `accessToken.claims` as opposed to `idToken.claims` (where we're getting `user` [above](https://github.com/CDCgov/prime-reportstream/pull/8208/files#diff-d10ebe8bd144dc544c78e474f801eb75d711a874b34f565b7c3cd6119848685aR60)), right?

Reply:
Also, VERY PEDANTIC POINT, but we could simplify this to a single iteration as opposed to at most three iterations.  This is a sub-microscopic point since our users shouldn't be assigned to more than a couple of Organization at most at any time so definitely not a problem :)

Reply:
Oooh yeah looks like the first point here is causing a critical bug:
<img width="595" alt="Screenshot 2023-02-08 at 21 27 50" src="https://user-images.githubusercontent.com/2421042/217701920-9075eb46-4fcb-4411-ad24-fd52bf1d8f5a.png">